src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const activeTabHandler = tabId => {
+    if (tabId !== activeTab) {
```

Comment:
You're comparing an object with another object using strict comparison. This could produce bugs because objects are compared by reference, not by value. Instead, compare the unique identifier of the objects. Here is how you can do it: ```jsx
if (tabId !== activeTab.id) {
```
This is a good practice to follow when comparing objects.